src/translations/indexMapper.js

```diff
@@ -0,0 +1,362 @@
+import { arrayFilter, arrayMap } from './../helpers/array';
```

Comment:
I can't find a good reason why the new index mappers are located in path `src/translations`? This can be easily confused with i18n.

I suggest renaming this directory to `src/indexMapper`

Reply:
It will be moved to the `dataMap` directory just after @budnix will finish his task about cell meta manager (https://github.com/handsontable/handsontable/pull/6254).